js/components/image.js

```diff
@@ -75,6 +80,10 @@ Fliplet.FormBuilder.field('image', {
 
         return Promise.reject('Please fill in required fields.');
       }
+      if (!this.isImageValid) {
```

Comment:
Empty line before block

```suggestion

      if (!this.isImageValid) {
```

Reply:
Fixed.